### PR TITLE
Updated summary panels in Dashboard

### LINKED ISSUE #8818: UX Unification – restyle dashboard summary metrics
### Is your proposal related to a problem?

Dashboard summary metrics look a bit dated.

### Describe the solution you'd like

Here is the proposed design for this component:

dashboard-header-latest

[Figma link](https://www.figma.com/proto/h67EsVXdWsfu38WGGxWfpi/Wagtail-Design-System?node-id=6087%3A36870&viewport=3489%2C1106%2C0.51&scaling=min-zoom&starting-point-node-id=6899%3A41010&show-proto-sidebar=1&hide-ui=1)

The goal is to reskin the component while keeping the implementation more or less the same. We could nonetheless consider:

- Adding a Storybook story for it
- Refactoring the markup so the `site_summary.html`’s `section` has a `aria-labelledby` pointing at the heading.
- Moving the CSS for this from `_home.scss` to its own component file
- Removing all usage of `float`, `clear`, and position relative/absolute from this code (#8125, #8126).

### PR DESCRIPTION
**Pull request by PaarthAgarwal**
Fixes #8818 
Before:

Screenshot from 2022-07-29 17-04-57

After:

Screenshot from 2022-07-30 15-29-47

Edit1: Updated Header Styling
Edit2: Realigned avatar icon

### CODE REVIEW

File: client/scss/components/_header.scss [hunk 1]
```diff
@@ -36,6 +37,11 @@
     font-weight: 400;
   }
 
+  .w-header__description {
+    font-size: 0.6em;
+    font-weight: 400;
+  }
```
Comment (lb-) [inline client/scss/components/_header.scss:43]: Can we try to use existing theme variables here?

File: wagtail/admin/templates/wagtailadmin/shared/header.html [hunk 2]
```diff
@@ -29,6 +29,9 @@ <h1 class="w-header__title">
                         <div class="w-header__glyph avatar"><img src="{{ avatar }}" alt="" /></div>
                     {% endif %}
                     {{ title }}{% if subtitle %} <span class="w-header__subtitle">{{ subtitle }}</span>{% endif %}
+                    {% if header_description %}
+                        <div class="w-header__description">{{ header_description }}</div>
+                    {% endif %}
                 </h1>
```
Comment (lb-) [inline wagtail/admin/templates/wagtailadmin/shared/header.html:35]: I think the description should be outside the h1 tag. 

Technically the subtitle should also - but that is a legacy decision we shouldn't change now. 

Remember that the h1 tag is not just a visual thing but a critical part of the page's context given to screen readers and other assistive technology. 

Adding a description of the page is useful but it's not really part of the h1 title.

File: client/scss/layouts/_home.scss [hunk 3]
```diff
@@ -46,22 +46,23 @@
...
-      margin-inline-end: 0.5em;
+      margin-inline-end: 0.15em;
     }
 
     a {
       position: relative;
       display: block;
       width: 100%;
       height: 100%;
+      color: theme('colors.secondary.DEFAULT');
```
Comment (lb-) [inline client/scss/layouts/_home.scss:57]:
```suggestion
      color: currentColor:
```

As discussed the other day, best to use the currentColor approach Here if possible as it avoids an additional colour setting and makes it easier to override the parent container styles and not have to also apply it to the link. 

Alternatively you could use `color: inherit;` to achieve the same goal.

File: wagtail/admin/templates/wagtailadmin/shared/header.html [hunk 4]
```diff
@@ -29,6 +29,9 @@ <h1 class="w-header__title">
                         <div class="w-header__glyph avatar"><img src="{{ avatar }}" alt="" /></div>
                     {% endif %}
                     {{ title }}{% if subtitle %} <span class="w-header__subtitle">{{ subtitle }}</span>{% endif %}
+                    {% if header_description %}
+                        <div class="w-header__description">{{ header_description }}</div>
+                    {% endif %}
```
Comment (lb-) [inline wagtail/admin/templates/wagtailadmin/shared/header.html:34]: A reminder to add the `description` to the variables the template accepts at the top of the docs above.

File: client/scss/components/_header.scss (hunk 1 above)
Comment (PaarthAgarwal) [reply client/scss/components/_header.scss:43]: I don't remember any variables for font size and weight. Most of the places I've looked at are using hardcoded values

**Review by lb- — COMMENTED** [review]
@PaarthAgarwal as per discussion, I ended up doing a bit of polish here to get this over the line.

* few issues with smaller breakpoints & the avatar
* using our theme variables where possible
* renaming `header_description` to `description` as other variables are not prefixed
* going a fair bit further with the summary panels as per original issue

**Comment by lb-** [comment]
Screen Shot 2022-07-31 at 1 07 15 am

With these changes - we will now have RTL support for the dashboard summary panel yay

**Pull request merged**